scripts/main/view.js

```diff
@@ -1054,6 +1054,9 @@ view.settings = {
 		 * @returns {void}
 		 */
 		setLogin: function () {
+			const hasToken = true;
+			const tokenBtnText = lychee.locale[hasToken ? "RESET_TOKEN" : "ENABLE_TOKEN"];
```

Comment:
`hasToken` is always true here.

If you don't feel like fixing it, I would at least recommend to remove this and also remove `ENABLE_TOKEN` from the locales as is never used right now.

I better approach would be as follows which mostly requires changes in the backend.

In `App\Models\User`

 * add `token` to the `$hidden` array
 * add `has_token` to the `$appends` array
 * write an accessor method `getHasTokenAttribute(): bool` which basically returns `$this->token !== null`

As the object of the authenticated user is nowadays sent to the frontend, you can then use `lychee.user.has_token` here in the frontend.